astroquery/mast/observations.py

```diff
@@ -112,9 +125,9 @@ def get_metadata(self, query_type):
         """
 
         if query_type.lower() == "observations":
-            colconf_name = "Mast.Caom.Cone"
+            colconf_name = ObservationsClass.caom_cone
```

Comment:
Don't you want these to be `self.caom_cone` (and similar) rather than `ObservationsClass.caom_cone`?